src/tests_exhaustive.c

```diff
@@ -13,6 +13,8 @@
 #define EXHAUSTIVE_TEST_ORDER 13
 #endif
 
+#define EXHAUSTIVE_TEST_CURVE_IS_EVEN_ORDERED (SECP256K1_B == 1 || SECP256K1_B == 6 || SECP256K1_B == 8)
```

Comment:
```suggestion
/* These values of B are all values in [1, 8] that result in a curve with even order. */ 
#define EXHAUSTIVE_TEST_CURVE_HAS_EVEN_ORDER (SECP256K1_B == 1 || SECP256K1_B == 6 || SECP256K1_B == 8)
```

Reply:
👍 Done.

Reply:
Oops, missed the second part of your recommendation. Agree that `_HAS_EVEN_ORDER` sounds better than `_IS_EVEN_ORDERED`. Force-pushed once again.